apps/src/templates/sectionAssessments/SectionAssessments.jsx

```diff
@@ -59,25 +59,34 @@ class SectionAssessments extends Component {
             scriptId={scriptId}
             onChange={this.onChangeScript}
           />
+          {!isLoading &&
+            <div>
+              <div style={{...h3Style, ...styles.header}}>
+                {i18n.selectAssessment()}
+              </div>
+              <AssessmentSelector
+                assessmentList={assessmentList}
+                assessmentId={assessmentId}
+                onChange={this.props.setAssessmentId}
+              />
+            </div>
+          }
         </div>
-        <div>
-          <div style={{...h3Style, ...styles.header}}>
-            {i18n.selectAssessment()}
+        {!isLoading &&
+          <div>
+            {/* Assessments */}
```

Comment:
Whoa! I didn't know you could include comments like that in the return of a render!